tests/utils/verification/volumeproperties.go

```diff
@@ -150,3 +150,13 @@ func VerifyDetachedStatus(name, hostName, esxName string) bool {
 	log.Printf("Timed out to poll status\n")
 	return false
 }
+
+// GetPolicyNameForVol returns the vsan policy name used by the volume using docker cli
+func GetPolicyNameForVol(hostname string, volName string) string {
```

Comment:
1. better home should be `tests/utils/dockercli` so the reference will be dockercli. GetPolicyNameForVol

2. how about renaming to `GetAssociatedPolicyName()`?